src/cim/components/ClusterDeployment/constants.ts

```diff
@@ -1,20 +1,18 @@
-import { ClusterDeploymentWizardStepsType } from './types';
+import { TFunction } from 'i18next';
 
-export const CLUSTER_HOSTS_SELECTION_LABELS: { [key in string]: string } = {
-  hostCount: 'Host Count',
-  masterLabels: 'Host Labels', // used in both masters-only and "auto-select masters" contexts
-  workerLabels: 'Worker Hosts Labels',
-};
+export const CLUSTER_HOSTS_SELECTION_LABELS = (t: TFunction) => ({
```

Comment:
I think the name should change. When we see `CLUSTER_HOSTS_SELECTION_LABELS` it looks like it's a constant, but it's a function.

Reply:
So do you suggest renaming all constants with (t:TFunction) to other names that look like function names?

Reply:
Done!